sourcegraph/sourcegraph
# Cody: Minior clean ups and papercut fixs

Just minor paper cut fixs and clean ups:
- Removed unused config `cody.enabled`
- Based on feedback from sqs, and beyang updated tooltip for non-embedded codebase to a more positive and informative message as the current one caused some [confusion for users](https://discord.com/channels/969688426372825169/1097918731260285008/1105036059840098346) 
- Update descriptions for all configurations 
- Check if access token value is valid before adding to secret storage https://github.com/sourcegraph/sourcegraph/issues/51573
- Add system notification for corrupted token back 😅 
- Display error messages on all views instead of just Chat view

### Before

<img width="487" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/d27d7acf-1e2e-4568-b121-1d6664003b88">

### After

Moved changes from https://github.com/sourcegraph/sourcegraph/pull/51970 to update the style for context title instead of codebase name
<img width="498" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/6e97ed04-b466-49d7-8b95-97861af92062">

<img width="493" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/454cebaf-970f-4b64-b395-13cc96ae1149">


### New

<img width="481" alt="Screenshot 2023-05-12 at 8 12 54 AM" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/500de4f3-5173-4fad-898c-84d246b0b8a1">

## Test plan



Nothing major was changed. All tests have passed.

Suggestions for testing:
- login logout
- use an invalid codebase to see the new info message
